WORKSPACE

```diff
@@ -131,6 +131,49 @@ rbe_autoconfig(
     use_checked_in_confs = False,
 )
 
+# RBE Autoconfig targets to do integration testing on the automatic toolchain
+# configs release process.
+load(
+    "//configs/tests/dependency-tracking:ubuntu1604_no_updates.bzl",
+    _ubuntu1604_bazel_no_updates = "bazel",
+    _ubuntu1604_configs_version_no_updates = "configs_version",
+    _ubuntu1604_digest_no_updates = "digest",
+    _ubuntu1604_registry_no_updates = "registry",
+    _ubuntu1604_repository_no_updates = "repository",
+)
+
+# Automatic config generation target for RBE Ubuntu 16.04
```

Comment:
change the comment as it reads the same as the one in line 124 and this has a different purpose

Reply:
Done